dex-it/src/test/scala/com/wavesplatform/it/sync/api/ws/WsAddressStreamTestSuite.scala

```diff
@@ -323,27 +327,39 @@ class WsAddressStreamTestSuite extends WsSuiteBase with TableDrivenPropertyCheck
         wsc.close()
       }
 
-      "NTF asset issued " - {
+      "NTF asset" - {
+
+        def validateBalances(wsc: WsConnection, wb: WsBalances, a: KeyPair, nft: Boolean = true): Unit = {
+          def assetBalance(asset: String, balance: Double = 1.0): (Asset, WsBalances) = Asset.fromString(asset).get -> WsBalances(balance, 0)
+
+          val nftMap = if (nft) wavesNode1.api.nftAssetsByAddress(a).map(a => assetBalance(a.assetId)) else Map.empty
 
-        def validateBalances(wsc: WsConnection, wb: WsBalances, a: KeyPair): Unit =
           assertChanges(wsc)(
-            (wavesNode1.api.assetsBalance(a).balances.map { b =>
-              Asset.fromString(b.assetId).get -> WsBalances(b.balance.toDouble, 0)
-            } ++ Map(Waves -> wb)).toMap
+            (wavesNode1.api.assetsBalance(a).balances.map(b => assetBalance(b.assetId, b.balance.toDouble))
+            ++ Map(Waves -> wb)
+            ++ nftMap).toMap
           )()
+        }
 
-        "via IssueTx should be in the address stream" in {
+        "issued via IssueTx" in {
 
           val acc = mkAccountWithBalance(10.waves -> Waves)
 
           broadcastAndAwait(mkIssue(acc, "testAssetNT", 1L, 0))
 
-          val wsc = mkWsAddressConnection(acc)
-          validateBalances(wsc, WsBalances(9, 0), acc)
-          wsc.close()
+          step("should be in the address stream")
+          val wsc1 = mkWsAddressConnection(acc)
+          validateBalances(wsc1, WsBalances(9, 0), acc)
+          wsc1.close()
+
+          step("shouldn't be in the filtered address stream")
+          val wsc2 = mkWsAddressFilteredConnection(acc, Set(WsAddressBalancesFilter.ExcludeNft))
+          validateBalances(wsc2, WsBalances(9, 0), acc, false)
```

Comment:
Also, please add the name of the parameter for `hasNft`. It is hard to read Boolean values, because we need to return to the declaration of the method again and again.
```scala
validateBalances(wsc2, WsBalances(9, 0), acc, hasNft = false)
```
Please do this below too.

Reply:
ok